docs/tutorials/ansible-napalm.rst

```diff
@@ -0,0 +1,217 @@
+
+      tasks:                                     #Begin Tasks
+        - name: get facts from device            #Task Name
+          napalm_get_facts:                      #Call the napalm module, in this case napal_get_facts
+            optional_args: {'secret': password}  #The enable password for Cisco IOS
+            hostname: "{{ inventory_hostname }}" #This is a parameter and is derived from your ansible inventory file
+            username: 'user'                     #The username to ssh with
+            dev_os: 'ios'                        #The hardware operating system
+            password: 'password'                 #The line level password
+            filter: 'facts'                      #The list of items you want to retrieve. The filter keyword is _inclusive_ of what you want
+          register: result                       #Ansible function for collecting output
+
+        - name: print results                    #Task Name
+          debug: msg="{{ result }}"              #Display the collected output
+
+
+Keeping with our example dir at the beginning of the Readme, we now have this layout::
+
+    user@host ~/workspace/ansible-playbooks
+    08:16 $ ls -la
+    total 32
+    drwxrwxr-x 3 user user 4096 Feb 26 07:24 .
+    drwxrwxr-x 8 user user 4096 Feb 25 16:32 ..
+    -rw-rw-r-- 1 user user  404 Feb 26 07:24 inventory.yaml
+
+
+You would run this playbook like as::
+
+  cd ~/workspace
+
+  ansible-playbook ansible-playbooks/inventory.yaml
+
+
+And it should produce output similar to this::
+
+
+    PLAY [Push config to switch group.] ********************************************
+
+    TASK [get facts from device] ***************************************************
+    ok: [192.168.0.11]
+
+    TASK [print results] *******************************************************************
+    ok: [192.168.0.11] => {
+        "msg": {
+            "ansible_facts": {
+                "facts": {
+                    "fqdn": "router1.not set",
+                    "hostname": "router1",
+                    "interface_list": [
+                        "FastEthernet0/0",
+                        "GigabitEthernet1/0",
+                        "GigabitEthernet2/0",
+                        "GigabitEthernet3/0",
+                        "GigabitEthernet4/0",
+                        "POS5/0",
+                        "POS6/0"
+                    ],
+                    "model": "7206VXR",
+                    "os_version": "7200 Software (C7200-ADVENTERPRISEK9-M), Version 15.2(4)S7, RELEASE SOFTWARE (fc4)",
+                    "serial_number": "0123456789",
+                    "uptime": 420,
+                    "vendor": "Cisco"
+                }
+            },
+            "changed": false
+        }
+    }
+
+    PLAY RECAP *********************************************************************
+    192.168.0.11               : ok=2    changed=0    unreachable=0    failed=0
+
+
+Copyright 2016-present Nike, Inc.
```

Comment:
Also, put it at the top. As it's commented out it shouldn't matter and it's the convention with code so let's do it with the doc as well.